lib/util.js

```diff
@@ -241,19 +236,12 @@ util = {
     },
 
     /**
-     * Get byte size from data.
-     * https://github.com/miktam/sizeof/blob/master/byte_size.js
-     * @param  {data}
-     * @return {Number} size
+     * Check whether a value is number-like
+     * https://github.com/lodash/lodash/issues/1148#issuecomment-141139153
+     * @return {Boolean}
      */
-    sizeOf: function(data) {
-       if (_.isString(data)) {
-          return data.length * ECMA_BYTE_SIZES.STRING;
-        } else if (_.isBoolean(data)) {
-          return ECMA_BYTE_SIZES.BOOLEAN;
-        } else if (_.isNumber(data)) {
-          return ECMA_BYTE_SIZES.NUMBER;
-        }
+    isNumeric: function (n) {
```

Comment:
There is no equivalent. https://github.com/lodash/lodash/issues/1148#issuecomment-141139153.